python/paddle/fluid/contrib/mixed_precision/bf16/amp_lists.py

```diff
@@ -83,7 +83,12 @@ def _update_list(self):
 bf16_initializer_list = {'fill_constant', 'uniform_random'}
 
 # always bf16
-bf16_list = {'elementwise_add', 'mul'}
```

Comment:
Is `elementwise_add` not working properly? Can we add this to this gray list?

Reply:
good catch, `elementwise_add` is working OK, it should be on gray list. The gray_list is imported to save space and contains `elementwise_add` already.